buildSrc/src/main/kotlin/ort-server-kotlin-jvm-conventions.gradle.kts

```diff
@@ -33,16 +35,22 @@ plugins {
     id("org.jetbrains.kotlin.jvm")
 }
 
-val javaVersion = JavaVersion.current()
-val maxKotlinJvmTarget = runCatching { JvmTarget.fromTarget(javaVersion.majorVersion) }
-    .getOrDefault(enumValues<JvmTarget>().max())
+java {
+    toolchain {
```

Comment:
If you take a toolchain into use, then the previous commit could have simply omitted setting up Java, as explained at https://github.com/oss-review-toolkit/ort/commit/494a32434823a3f0190db4e8e128b2efb223a14a.

Reply:
And note that also the Java version used to run the Gradle build itself needs to be configured, as the build uses ORT's Detekt rules which are also built with Java 21, see https://github.com/oss-review-toolkit/ort-workbench/commit/b800aaac9fc2c36b3246f579f1ed5a48b712ef02.

Reply:
Thanks, this should simplify the workflows.